packages/react-instantsearch/src/components/RefinementList.test.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import Enzyme, { mount } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
-Enzyme.configure({ adapter: new Adapter() });
-
 import RefinementList from './RefinementList';
 
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
not for this PR, but maybe it should be done globally and overridden if we test another version?